Makefile

```diff
@@ -255,7 +255,7 @@ install-sdk:
 
 .PHONY: install-tools
 install-tools:
-	@${GO_FLAGS} go get -u \
```

Comment:
How does this affect CI, the downloaded tools are not present in the filesystem.

In the main repository we faced an issue with inconsistent tools version. The CI had always new version but local version ware outdated due to missing `-u`.

Reply:
CI will always download the tool. The `-u` is only about updating the libraries that are being used. I thought this meant *also* the ones specified in the current directory's `go.mod`/`go.sum`, but I was wrong. The doc says:

> The -u flag instructs get to use the network to update the named packages and their dependencies. By default, get uses the network to check out missing packages but does not use it to look for updates to existing packages.

The purpose of this PR is to get the builds to not change go.mod/go.sum, unless explicitly told to do so.